### ISSUE TITLE
求助 Remote socket error: unable to write

### ISSUE THREAD

**Issue by shawngao** [shadowsocks/libQtShadowsocks#85]
按照教程配置好的，手机端也可以连上，默认设置全局模式下手机应用上网和墙内网页都OK（时不时出现Remote socket error: Connection refused但不影响访问）。可是手机浏览器访问墙外的youtube和facebook服务器端就显示这个Remote socket error: Unable to write，手机端网页返回 ERR_NAME_NOT_RESOLVED，目测好像是DNS解析失败，可是服务器主机自己的浏览器DNS解析和访问墙外网页都没问题，手机通过ss的访问网页DNS解析不应该在服务器端么？搞不懂了，求忙帮指导，谢谢~

Connecting 8.8.8.8:53 from 223.104.38.18:7753
Connecting 8.8.8.8:53 from 223.104.38.18:7754
Connecting 8.8.8.8:53 from 223.104.38.18:7755
Connecting 8.8.8.8:53 from 223.104.38.18:7756
Remote socket error: Unable to write
Connecting 8.8.8.8:53 from 223.104.38.18:7757
Connecting 8.8.8.8:53 from 223.104.38.18:7758
Connecting 8.8.8.8:53 from 223.104.38.18:7759
Connecting 8.8.8.8:53 from 223.104.38.18:7760
Connecting 8.8.8.8:53 from 223.104.38.18:7761
Remote socket error: Unable to write
Connecting 8.8.8.8:53 from 223.104.38.18:7762
Connecting 36.110.144.74:8010 from 223.104.38.18:7763

**Comment by librehat** [comment]
So you're using `shadowsocks-libqss` as your sever, and shadowsocks for Android on your phone?

**Comment by shawngao** [comment]
@librehat  yes, exactly. That's why i came here for help. I'm using the latest release shadowsocks-libqss win64 v1.9.0. Config as below
{
    "server":"0.0.0.0",
    "server_port":8388,
    "local_address":"127.0.0.1",
    "local_port":1088,
    "password":"password",
    "timeout":600,
    "method":"chacha20",
    "http_proxy": false,
    "auth": false
}
port 8388 is available for remote connecting and connection is ok by shadowsocks on android.
APPs like wechat is well functional.
Problem is in web browser, it can't resolve url even www.baidu.com, not to mention youtube.com ...
_(:3」∠)_ cloud u please help me dealing with this, thx a lot...

**Comment by ichengzi** [comment]
My android has this problem also, can anyone help solve this? And my android dns is set to `114.114.114.114`
